apps/sms/test/marionette/lib/views/inbox/accessors.js

```diff
@@ -0,0 +1,37 @@
+var SELECTORS = Object.freeze({
+  main: '#thread-list',
+  firstConversation: '.threadlist-item',
+  smsConversation: '.threadlist-item[data-last-message-type="sms"]',
+  mmsConversation: '.threadlist-item[data-last-message-type="mms"]',
+  navigateToComposerHeaderButton: '#threads-composer-link'
+});
+
+module.exports = {
+  get firstConversation() {
+    return client.helper.waitForElement(
+      SELECTORS.firstConversation
+    );
+  },
+
+  get smsConversation() {
+    return client.helper.waitForElement(
+      SELECTORS.smsConversation
+    );
+  },
+
+  get mmsConversation() {
+    return client.helper.waitForElement(
+      SELECTORS.mmsConversation
+    );
+  },
+
+  waitToAppear: function() {
+    return client.helper.waitForElement(SELECTORS.main);
+  },
+
+  navigateToComposer: function() {
```

Comment:
You mention that navigation methods should reside in view class... or it's temporal and we should change it in the follow-ups?